hazelcast/include/hazelcast/client/query/predicates.h

```diff
@@ -64,6 +64,15 @@ namespace hazelcast {
                     out_stream.write<int32_t>(static_cast<int32_t>(sizeof...(values)));
                     out_stream.write_objects(values...);
                 }
+
+                template<typename T>
+                multi_predicate(const std::string attribute_name, hazelcast_client &client, const std::vector<T> &values) : base_predicate(client) {
+                    out_stream.write(attribute_name);
+                    out_stream.write<int32_t>(static_cast<int32_t>(values.size()));
+                    for (const T &value : values) {
```

Comment:
won't compile when T is `bool`

```suggestion
                    for (std::vector<T>::const_reference value : values) {
```

Reply:
sorry, my mistake. it does compile. Doesn't compile when reference isn't const.

Reply:
I think that the suggestion is the same as the existing one, do you really want the suggestion applied?